index.js

```diff
@@ -28,12 +28,13 @@ module.exports = {
             scope: SETTINGS.OAUTH_SCOPES,
             apiNamespace: 'v2' // URL suffix (after host)
         };
-
+        //TODO define waterbutlerUrl for all other environments
         if (BACKEND === 'local') {
             ENV.OSF.url = 'http://localhost:5000/';
```

Comment:
Staging is:
`https://staging-files.osf.io/`
stage2 is:
`https://staging2-files.osf.io/`
prod is:
`https://files.osf.io/`